### ISSUE TITLE
sklearn models

### ISSUE THREAD

**Issue by evanfeinberg** [pandegroup/deepchem#65]
In standard.py, change model_name --> model_instance, and change for loop to simply self.raw_model = model_instance. This is to enable the user to take advantage of the wide array of models available in sklearn.

**Comment by evanfeinberg** [comment]
With a refactoring that has been pushed to the latest WIP, I believe this feature has been implemented. The user is now encouraged to directly create an instance of a class to act as the model, rather than using ```model_builder()```, which I tentatively suggest we should delete and deprecate. The user should now be able to use any ```deepchem``` or ```sklearn``` model type that they would like. This is passing all nosetests. Let's discuss merging offline.

**Comment by rbharath** [comment]
This has been merged in #82. Nice work with this!

**Comment by evanfeinberg** [comment]
Thank you!
